core/src/main/java/com/graphhopper/reader/dem/ElevationInterpolator.java

```diff
@@ -0,0 +1,99 @@
+package com.graphhopper.reader.dem;
+
+import com.graphhopper.util.PointList;
+
+public class ElevationInterpolator {
+
+	public static final double EPSILON = 0.00001;
+	public static final double EPSILON2 = EPSILON * EPSILON;
+
+	public double calculateElevation(double lat, double lon, double lat0, double lon0, double ele0, double lat1,
```

Comment:
It should be more clear what this method does. Especially as the parameters are somehow unexpected. Add a comment here?